Client/Frontend/Browser/Tabs/InactiveTabItemCell.swift

```diff
@@ -0,0 +1,134 @@
+
+        NSLayoutConstraint.activate([
+            containerView.topAnchor.constraint(equalTo: contentView.topAnchor),
+            containerView.bottomAnchor.constraint(equalTo: contentView.bottomAnchor),
+            containerViewLeadingConstraint,
+            containerView.trailingAnchor.constraint(equalTo: contentView.trailingAnchor),
+
+            leftImageView.heightAnchor.constraint(equalToConstant: InactiveTabItemCellModel.UX.ImageSize),
+            leftImageView.widthAnchor.constraint(equalToConstant: InactiveTabItemCellModel.UX.ImageSize),
+            leftImageView.leadingAnchor.constraint(equalTo: containerView.leadingAnchor,
+                                                   constant: InactiveTabItemCellModel.UX.ImageViewLeadingConstant),
+            leftImageView.centerYAnchor.constraint(equalTo: midView.centerYAnchor),
+            leftImageView.topAnchor.constraint(greaterThanOrEqualTo: containerView.topAnchor,
+                                               constant: InactiveTabItemCellModel.UX.ImageTopBottomMargin),
+            leftImageView.bottomAnchor.constraint(lessThanOrEqualTo: containerView.bottomAnchor,
+                                                  constant: InactiveTabItemCellModel.UX.ImageTopBottomMargin),
+
+            midView.topAnchor.constraint(equalTo: containerView.topAnchor),
+            midView.bottomAnchor.constraint(equalTo: containerView.bottomAnchor),
+            midView.leadingAnchor.constraint(equalTo: leftImageView.trailingAnchor,
+                                             constant: InactiveTabItemCellModel.UX.MidViewLeadingConstant),
+            midView.trailingAnchor.constraint(equalTo: containerView.trailingAnchor,
+                                              constant: InactiveTabItemCellModel.UX.MidViewTrailingConstant),
+
+            titleLabel.topAnchor.constraint(equalTo: midView.topAnchor,
+                                            constant: InactiveTabItemCellModel.UX.LabelTopBottomMargin),
+            titleLabel.bottomAnchor.constraint(equalTo: midView.bottomAnchor,
+                                               constant: -InactiveTabItemCellModel.UX.LabelTopBottomMargin),
+            titleLabel.leadingAnchor.constraint(equalTo: midView.leadingAnchor),
+            titleLabel.trailingAnchor.constraint(equalTo: midView.trailingAnchor),
+
+            bottomSeparatorView.heightAnchor.constraint(equalToConstant: InactiveTabItemCellModel.UX.SeparatorHeight),
+            bottomSeparatorView.bottomAnchor.constraint(equalTo: containerView.bottomAnchor),
+            bottomSeparatorView.leadingAnchor.constraint(equalTo: titleLabel.leadingAnchor),
+            bottomSeparatorView.trailingAnchor.constraint(equalTo: containerView.trailingAnchor),
+        ])
+
+        leftImageView.setContentHuggingPriority(.required, for: .vertical)
+
+        selectedBackgroundView = selectedView
+        applyTheme()
+    }
+
+    func applyTheme() {
```

Comment:
Do we call applyTheme on theme change? We normally use notifiable protocol. I don't see it so maybe you call it from elsewhere?